packages/forms-web-app/src/dynamic-forms/dynamic-components/site-address/question.js

```diff
@@ -176,7 +176,7 @@ class SiteAddressQuestion extends Question {
 			answer.postcode
 		];
 
-		return addressComponents.filter(Boolean).join(', ');
+		return addressComponents.filter(Boolean).join('<br>');
```

Comment:
looks like this needs escaping
try using <b>test</b> as a component, need the line breaks but not the bold, can use the nl2br filter in view or escape the data here